### ISSUE TITLE
App config can't be updated using from_envvar

### ISSUE THREAD

**Issue by cisarpavel** [ergoithz/browsepy#20]
When using uwsgi with BROWSEPY_SETTINGS=/some/path/app.cfg, from_envvar ignores to update all lowercase variables. Only uppercase VARS can be updated this way, so none of
```
    directory_base=compat.getcwd(),
    directory_start=compat.getcwd(),
    directory_remove=None,
    directory_upload=None,
    directory_tar_buffsize=262144,
    directory_downloadable=True,
    use_binary_multiples=True,
    plugin_modules=[],
    plugin_namespaces=(
        'browsepy.plugin',
        'browsepy_',
        '',
        ),
    exclude_fnc=None,
```

can be updated using cfg file. It's standard behaviour of from_envvar.

**Comment by ergoithz** [comment]
Thanks for your report.

**Comment by ergoithz** [comment]
Fixed in 0.5.4
